src/rpc_server.rs

```diff
@@ -63,13 +64,14 @@ pub struct GetPriorityFeeEstimateOptions {
     pub recommended: Option<bool>, // return the recommended fee (median fee excluding vote txns)
 }
 
-#[derive(Serialize, Clone)]
+#[derive(Serialize, Clone, Debug, Default)]
 #[serde(rename_all(serialize = "camelCase", deserialize = "camelCase"))]
 pub struct GetPriorityFeeEstimateResponse {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub priority_fee_estimate: Option<f64>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub priority_fee_levels: Option<MicroLamportPriorityFeeEstimates>,
+    pub params: GetPriorityFeeEstimateRequest,
```

Comment:
Biggest change here is to return the parameters back to client to see what we used for calculation

Reply:
I don't think this should be in the response. We can return an informative error if deserialization fails, but I think thats where we should draw the line